moveit_core/robot_state/include/moveit/robot_state/attached_body.h

```diff
@@ -110,6 +111,47 @@ class AttachedBody
     return attach_trans_;
   }
 
+  /** \brief Get the fixed transform to a named frame on this body (not a transform to a visual or collision shape) */
+  const Eigen::Affine3d& getFixedTransform(const std::string& frame_name) const
```

Comment:
Why introducing an alias for getNamedTransform?

Reply:
I probably imagined using it like `getFrameTransform` in `robot_state.cpp` to look in both the named frames and the shapes' frames, but I don't think it will be needed. 

It probably came to my mind because `getFixedTransforms` is a bit of an awkward name. Maybe `getGeometryTransforms` or `getShapeTransforms` might be better?

Reply:
I couldn't follow my reasoning and removed the alias. It was unused.